Makefile.in

```diff
@@ -354,6 +354,13 @@ runtest-upstream:
 	cp $(stage2_prefix)/lib/ocaml/compiler-libs/*.cma _runtest/compilerlibs
 	cp $(stage2_prefix)/lib/ocaml/compiler-libs/*.a _runtest/compilerlibs
 	cp $(stage2_prefix)/lib/ocaml/compiler-libs/*.cmxa _runtest/compilerlibs
+	cp _build2/default/backend/debug/dwarf/dwarf_flags/dwarf_flags.cma \
+	cp _build2/default/backend/debug/dwarf/dwarf_ocaml/dwarf_ocaml.cma \
+	  _runtest/compilerlibs
+	cp _build2/default/backend/debug/dwarf/dwarf_high/dwarf_high.cma \
+	  _runtest/compilerlibs
+	cp _build2/default/backend/debug/dwarf/dwarf_low/dwarf_low.cma \
+	  _runtest/compilerlibs
```

Comment:
Hmm, I'm guessing we just forgot to add these ones in the previous PRs?  Also, we need the `.a` and `.cmxa` files, no?

Reply:
I removed those lines and added the missing dependencies to the correct place in the main dunefile